.github/workflows/ci.yml

```diff
@@ -815,6 +815,7 @@ jobs:
                 -DBUILD_SHARED_LIBS=ON                          \
                 -DCODE_COVERAGE=${{ env.CODE_COVERAGE }}        \
                 -DCMAKE_INSTALL_PREFIX=$CPPINTEROP_DIR          \
+                -DCMAKE_COMPILE_WARNING_AS_ERROR=ON            \
```

Comment:
I believe having `-DLLVM_ENABLE_WERROR=On` will set up everything that we need in cmake.

Reply:
I will switch the flag and revert the cmake minimum version change.